Minor changes (#1)
tolusha/che-docs: modules/installation-guide/partials/proc_setting-up-the-keycloak-che-username-readonly-theme-for-the-eclipse-che-login-page.adoc

// Module included in the following assemblies:
//
// configuring-che

//ID versions:

// setting-up-the-keycloak-che-username-readonly-theme-for-the-eclipse-che-login-page

// setting-up-the-{identity-provider-id}-{prod-id-short}-username-readonly-theme-for-the-{prod2-id}-login-page-for-the-{prod2-id}-login-page

[id="setting-up-the-keycloak-che-username-readonly-theme-for-the-eclipse-che-login-page_{context}"]

> = Setting up the {identity-provider} {prod-id-short}-username-readonly theme for the {prod2} login page

Reviewer: Let's avoid `prod2`, too. What is the reason for having such attributes?

Author: In some cases, I needed to have Codeready Workspaces and Eclipse Che.

I can make this proc work without it.

Reviewer: We have `prod-short` -- what's wrong with using "Che" and "CodeReady Workspaces"?

> The following procedure is relevant for all {prod-short} instances with the enabled OpenShift OAuth service.

Reviewer: ```suggestion
The following procedure is relevant for all {prod-short} instances with the OpenShift OAuth service enabled.
```

> After a user log in to {identity-provider} for the first time using the OpenShift credentials, a page allowing the user to update the account information displays. The user can change his username that was automatically created during the first login, but then a change to the Keycloak settings need to be done. Keeping the changed username without the steps mention below will prevent the user's workspaces from running.

Reviewer: ```suggestion
When a user logs in to {identity-provider} for the first time using OpenShift credentials, a page allowing the user to update account information is displayed. It is possible to change the username that was automatically created during the first login, but then {identity-provider} settings must be modified. Failing to modify {identity-provider} settings prevents the user's workspaces from running.
```

.Prerequisites

* An instance of {prod-short} running in OpenShift.
* A user is logged in to the {identity-provider} service.

.Procedure

> After changing a username, set the Login Theme option to `readonly`.

Reviewer: ```suggestion
After changing a username, set the *Login Theme* option to `readonly`.
```

> . In the main *Configure* menu on the left, select *Realm Settings*

Reviewer: ```suggestion
. In the main *Configure* menu on the left, select *Realm Settings*:
```

+
image::keycloak/{project-context}-keycloak-username-readonly-theme.png[link="../_images/keycloak/{project-context}-keycloak-username-readonly-theme.png"]

. Navigate to the *Themes* tab.

> . In the *Login Theme* field, select the `{prod-id-short}-username-readonly` option and click btn:[Save] to apply the changes.

Reviewer: ```suggestion
. In the *Login Theme* field, select the `{prod-id-short}-username-readonly` option and click the btn:[Save] button to apply the changes.
```
